conf/config-default.yaml

```diff
@@ -555,6 +556,9 @@ plugin_attr:
       send: 60s
 #  redirect:
 #    https_port: 8443   # the default port for use by HTTP redirects to HTTPS
+  inspect:
+    delay: 3            # in seconds
+    hooks_file: "/var/run/apisix_inspect_hooks.lua"
```

Comment:
What is this file for? Has Lua code in it?

Reply:
yes, this hooks file is used to define breakpoints, which itself is lua file. When the administrator needs to set breakpoints to inspect something, he would edit this file, or link another file to this path.

Reply:
Normally this file is non-exist. It's only available when you need to set breakpoints.

Reply:
There are security risks.
This Lua code file must be in the APISIX directory, and users who do not have write permissions to the APISIX directory cannot write or modify this file.